carbonmark-api/src/app.constants.ts

```diff
@@ -6,11 +6,20 @@ const GRAPH_API_ROOT = "https://api.thegraph.com/subgraphs/name";
 /** Graph URLS */
 /** Note: the keys of the below objects are used when selecting gql files for type generation */
 export const GRAPH_URLS = {
-  marketplace: `${GRAPH_API_ROOT}/najada/marketplace-matic`,
-  assets: `${GRAPH_API_ROOT}/cujowolf/klima-refi-current-holdings`,
-  offsets: `${GRAPH_API_ROOT}/klimadao/polygon-bridged-carbon`,
-  tokens: `${GRAPH_API_ROOT}/klimadao/klimadao-pairs`,
-  digitalCarbon: `${GRAPH_API_ROOT}/klimadao/polygon-digital-carbon`,
+  polygon: {
+    marketplace: `${GRAPH_API_ROOT}/najada/marketplace-matic`,
+    assets: `${GRAPH_API_ROOT}/cujowolf/klima-refi-current-holdings`,
+    offsets: `${GRAPH_API_ROOT}/klimadao/polygon-bridged-carbon`,
+    tokens: `${GRAPH_API_ROOT}/klimadao/klimadao-pairs`,
+    digitalCarbon: `${GRAPH_API_ROOT}/klimadao/polygon-digital-carbon`,
+  },
+  mumbai: {
+    marketplace: `${GRAPH_API_ROOT}/cujowolf/carbonmark-mumbai`,
+    assets: `${GRAPH_API_ROOT}/cujowolf/klima-refi-current-holdings`,
+    offsets: `${GRAPH_API_ROOT}/klimadao/polygon-bridged-carbon`,
```

Comment:
I like this approach but this is a problem, these graphs don't exist for mumbai (because many of the contracts don't exist on mumbai either), and probably won't for a long time.

Another challenge is that, even though I originally wanted mumbai marketplace subgraph to match the mainnet one, in reality there are some differences, and as they continue to develop/ship Marketplace features, there might be moments of divergence where changes make it to testnet before mainnet.

In my PR trying to get the frontend working in a basic state, I ended up just hot swapping and maintaining two sets of types, which also sucks, but has the benefit of explicitly exposing type differences to the developer...